graphics/tinygl/get.cpp

```diff
@@ -674,7 +674,9 @@ void GLContext::gl_get_pname(TGLenum pname, union uglValue *data, eDataType &dat
 	case TGL_SCISSOR_BOX:
 		// fall through
 	case TGL_SCISSOR_TEST:
-		error("gl_get_pname: TGL_SCISSOR_x option not implemented");
+		// error("gl_get_pname: TGL_SCISSOR_x option not implemented");
```

Comment:
This looks wrong, previously this error fired for both `TGL_SCISSOR_TEST` and `TGL_SCISSOR_BOX` via fall through.  This implements scissor test but `TGL_SCISSOR_BOX` is not implemented, so the error should be moved up to the `TGL_SCISSOR_BOX` case.